### ISSUE TITLE
no data received with kinect v1 

### ISSUE THREAD

**Issue by arpit15** [ros-drivers/freenect_stack#32]
I am able to see the depth and rgb image with libfreenect library. 
I am trying to use this package with ros kinetic, ubuntu 16.04 and kinect v1 with the following command.
`roslaunch freenect_launch freenect.launch`
However I am getting the [errors](https://gist.github.com/arpit15/712f730537eefeb21a79d78954324ce0) messages. Please let me know if I am using the correct commands or is there any other problem.

**Comment by vfdev-5** [comment]
Looking at your "errors" messages, you do not have any errors. The message "Stopping device RGB and Depth stream flush." can be also related with that there are no subscribers listening to the topic. 
Check with `rostopic hz /camera/rgb/image_color` whether you receive messages. What do you have when you call `rostopic list` ?

**Comment by arpit15** [comment]
Output of rostopic hz /camera/rgb/image_color
`subscribed to [/camera/rgb/image_color]
no new messages
no new messages
no new messages`

Output of rostopic list
[rostopics](https://gist.github.com/arpit15/f19c8c466e366ac6bdd3d5580f31e6c8)

**Comment by vfdev-5** [comment]
What is your kinect v1 model 1414 or 1417 ? Did you install `libfreenect` using `apt-get` or you have only `ros-kinetic-libfreenect`, what do you have on `locate libfreenect` ?

**Comment by arpit15** [comment]
Kinect is v1 1414.
I installed both. The output for `locate libfreenect`
[output](https://gist.github.com/arpit15/6f96955324f732843d261b53aae6826c)
full log of `roslaunch freenect_launch freenect.launch`
[log](https://gist.github.com/arpit15/e90a973820fc780f3a950d5a1e61b15b)

**Comment by vfdev-5** [comment]
Right, this can be the problem if you have multiple drivers of different versions. In my case I had a similar issue few weeks ago. Solution was to remove all `libfreenect` drivers from the system `/usr/lib`, `/usr/local/lib` etc. And install only `ros-kinetic-freenect-stack` which has `ros-kinetic-libfreenect` as dependency. The latter is a `libfreenect` driver version 0.5.1.

**Comment by sowrabhnayak** [comment]
I have connected Kinect V2 on my Ubuntu 14.04 which has got Indigo.
I have a different problem.... I'm not getting any errors but the system is not detecting the connected kinect.
some of the statements on running the command is 

 * /camera/driver/depth_registration: False
 * /camera/driver/enable_depth_diagnostics: False
 * /camera/driver/enable_ir_diagnostics: False
 * /camera/driver/enable_rgb_diagnostics: False
 * /camera/driver/rgb_camera_info_url: 
 * /camera/driver/rgb_frame_id: camera_rgb_optica...
 * /rosdistro: indigo
 * /rosversion: 1.11.21

could anyone of u please explain what the problem is...

And how do I make use of this data in the program in code that I write in Python.
Please explain me the procedures as well....

Thank you
